tests/test_random_number.py

```diff
@@ -0,0 +1,5 @@
+from src.random_number import random_number
+
+def test_random_number() -> None:
+    t = random_number(0, 100)
```

Comment:
Dai un nome più "parlante" alla variabile `t`.
In questo contesto è ovviamente semplice da capire cosa contiene ma non è sempre così.

Reply:
Definisci delle variabili costanti che contengano 0 e 100.